src/lidar_subscriber.cpp

```diff
@@ -46,6 +46,73 @@ static void merge_landmarks(Observation2D &first_landmark, Observation2D &last_l
   last_landmark.bearing_rad = first_landmark.bearing_rad*weight1+(last_landmark.bearing_rad)*weight2;
 }
 
+void visualize_landmarks(std::queue<Point2D> landmarks, rclcpp::Publisher<visualization_msgs::msg::Marker>::SharedPtr landmark_visualization_publisher){
+  auto message = visualization_msgs::msg::Marker();
+  rclcpp::Clock clock = rclcpp::Clock();
+  message.header.stamp = clock.now();
+  message.header.frame_id = "map";
+  message.action = 0;
+  message.type = 8;
+  message.ns = "landmark_list";
+  message.id = 0;
+  message.pose.orientation.w = 1.0;
+  message.scale.x = 0.04;
+  message.scale.y = 0.04;
+  auto color = std_msgs::msg::ColorRGBA();
+  color.r = 255;
```

Comment:
These hard constants should be eliminated in the code (including below on line 129) and stored somewhere else (probably in the header file). If we reach a critical mass of configuration options we might want to start looking into json files as an option to store settings.